website/client/components/challenges/challengeItem.vue

```diff
@@ -1,187 +1,270 @@
 <template lang="pug">
-.card
-  .row
-    router-link.col-12(:to="{ name: 'challenge', params: { challengeId: challenge._id } }")
-      h3(v-markdown='challenge.name')
-  .row
-    .col-6
-      div.details
-        span
-          .svg-icon.member-icon(v-html="icons.memberIcon")
-        span {{challenge.memberCount}}
-        // @TODO: Add in V2
-          span
-            .svg-icon.calendar-icon(v-html="icons.calendarIcon")
-          span
-            strong End Date:
+.challenge
+  .challenge-prize
+    .number
+      span.svg-icon(v-html="icons.gemIcon")
+      span.value {{challenge.prize}}
+    .label {{ $t('challengePrize') }}
+  .challenge-header
+    router-link(
+      :to="{ name: 'challenge', params: { challengeId: challenge._id } }"
+    )
+      h3.challenge-title(v-markdown='challenge.name')
+    .owner(v-if="fullLayout")
+      .owner-item
+        strong {{ $t('createdBy') }}:
+        user-link.mx-1(:user="challenge.leader")
+      .owner-item(v-if="challenge.group && challenge.group.name !== 'Tavern'")
+        strong {{ $t(challenge.group.type) }}:
+        group-link.mx-1(:group="challenge.group")
+    .meta
+      .meta-item
+        .svg-icon.user-icon(v-html="icons.memberIcon")
+        span.mx-1 {{challenge.memberCount}}
+      // @TODO: add end date
+        .meta-item
+          .svg-icon(v-html="icons.calendarIcon")
+          strong.mx-1 {{ $t('endDate')}}:
           span {{challenge.endDate}}
-      div.tags
-        span.tag(v-for='tag in challenge.tags') {{tag}}
-    .col-6.prize-section
-      div
-        span.svg-icon.gem(v-html="icons.gemIcon")
-        span.prize {{challenge.prize}}
-      div Challenge Prize
-  .row.description
-    .col-12
-      | {{challenge.summary}}
-  .well.row
-    .col-3
-      .count-details
-        .svg-icon.habit-icon(v-html="icons.habitIcon")
-        span.count {{challenge.tasksOrder.habits.length}}
-      div {{$t('habit')}}
-    .col-3
-      .count-details
-        .svg-icon.daily-icon(v-html="icons.dailyIcon")
-        span.count {{challenge.tasksOrder.dailys.length}}
-      div {{$t('daily')}}
-    .col-3
-      .count-details
-        .svg-icon.todo-icon(v-html="icons.todoIcon")
-        span.count {{challenge.tasksOrder.todos.length}}
-      div {{$t('todo')}}
-    .col-3
-      .count-details
-        .svg-icon.reward-icon(v-html="icons.rewardIcon")
-        span.count {{challenge.tasksOrder.rewards.length}}
-      div {{$t('reward')}}
+  .categories
+    span.category-label.category-label-blue(v-if='isLeader(challenge)') {{ $t('owned') }}
+    span.category-label(v-for='category in challenge.categories', :class="{'category-label-purple':isOfficial(category)}") {{ $t(category.name) }}
+  .challenge-description {{challenge.summary}}
+  .well-wrapper(v-if="fullLayout")
+    .well
+      div(:class="{'muted': challenge.tasksOrder.habits.length === 0}")
```

Comment:
could this be just generated from an array? then everything under `.number` and `.label` could be binded with `v-once` to speed up everything a bit